testsuite/model/pom.xml

```diff
@@ -118,6 +124,10 @@
             <artifactId>infinispan-component-annotations</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
```

Comment:
it is already defined here: https://github.com/keycloak/keycloak/blob/fed804160befe60b283b6d9c9ae26218902a79dc/testsuite/model/pom.xml#L289-L294

Reply:
I removed it from there as more profiles use it now. I don't think we need to exclude it from profiles that do not use it but let me know if you think otherwise.